prometheus_metrics/src/metrics.rs

```diff
@@ -712,7 +738,7 @@ impl Metrics {
             tick_delay_times: GaugeVec::new(
                 opts!("TICK_DELAY_TIMES", "Tick delay times"),
                 &["tick"],
-            )?,
+            )?, 
```

Comment:
linter?

Reply:
sorry, i will run `fmt` before commit, but it should be a CI for this task, isn't it?